onmt/inputters/dataset_base.py

```diff
@@ -130,3 +115,34 @@ def _construct_example_fromlist(self, data, fields):
             else:
                 setattr(ex, name, val)
         return ex
+
+
+# this is just temporary until the TextDatabase can be unified with the others
+class AudioVisualDataset(DatasetBase):
+    """
```

Comment:
Even if it's temporary AudioVisual is not really good (not handling movies) so far we used Audio and Img.
Anyway, shall we wait for the unified class before merging ?